public/js/Histogram.js

```diff
@@ -12,21 +16,28 @@ _.mixin(require('lodash-deep'));
  * @param {array} featureNames - An array of the features used in this screen.
  */
 function Histogram(sampleData, element, featureNames) {
+    var self = this;
+
     this.sampleData = sampleData;
     this.featureNames = featureNames;
     this.feature = 0;
-
     this.element = element;
-    this.fullWidth = 500;
-    this.fullHeight = 400;
+
+    this.fullWidth = $(this.element).width();
```

Comment:
It's a jQuery object that's been stored into a variable. If you're going to select an element more than once, it's good practice to cache the selector into a variable. That way, jQuery doesn't need to traverse the DOM to find the element again. It's stored in a variable so you can access it again immediately.

The $ is there just to show it's a jQuery object -- that's a convention set out in the AirBnb style guide! https://github.com/airbnb/javascript#jquery